reciprocalspaceship/dataset.py

```diff
@@ -539,16 +548,21 @@ def hkl_to_observed(self, m_isym="M/ISYM", inplace=False):
             dataset = self.copy()
 
         hkls = dataset.get_hkls()
-        isym = (dataset[m_isym] % 256).to_numpy() # Convert M/ISYM to ISYM
-        observed_hkls = hkl_to_observed(hkls, isym, dataset.spacegroup)
 
-        # Update HKLs
+        # GH#3: Separate combined M/ISYM into M and ISYM
+        isym = (dataset[m_isym] % 256).to_numpy()
+        dataset["PARTIAL"] = (dataset[m_isym]/256).astype(int) != 0
+        dataset.drop(columns=m_isym, inplace=True)
+        
+        # Update HKLs        
+        observed_hkls = hkl_to_observed(hkls, isym, dataset.spacegroup)
         index_keys = dataset.index.names
         dataset.reset_index(inplace=True)
         dataset[["H", "K", "L"]] = observed_hkls
         dataset[["H", "K", "L"]] = dataset[["H", "K", "L"]].astype("HKL")
         dataset.set_index(index_keys, inplace=True)
-
+        dataset.sort_index(inplace=True)
```

Comment:
Is there a reason we need to sort the indices here? Preserving the order leaves the option open for copying over columns from a version of the `Dataset` that has a numerical index (I think). This may not be a very important use case, but I wanted to make sure this won't break anything.

Reply:
I agree this sort is unnecessary -- I'll remove it